tests/plugins/easyimage/manual/selectionafterpaste.md

```diff
@@ -0,0 +1,14 @@
+@bender-tags: 4.9.0, bug, 932
+@bender-ui: collapsed
+@bender-ckeditor-plugins: sourcearea, wysiwygarea, floatingspace, toolbar, easyimage, undo
+
+1. Drag and drop some random images into editor
```

Comment:
Make sure that manual test steps are dead simple. Ask to drop explicitly two images.